sorted_db/db.go

```diff
@@ -136,6 +136,13 @@ func indexByte(s []byte, i, m int, c byte) int {
 	return -1
 }
 
+// Copies all bytes in s to a new destination buffer
+func makeCopy(s []byte) []byte {
```

Comment:
That's not really how Go works:

1. Comment required for Go Doc
2. Go style prefers short names: https://www.elastic.co/guide/en/elasticsearch/guide/current/_search_options.html#search-type